services/hanke-service/src/integrationTest/kotlin/fi/hel/haitaton/hanke/application/ApplicationControllerITest.kt

```diff
@@ -64,7 +68,7 @@ class ApplicationControllerITest(@Autowired override val mockMvc: MockMvc) : Con
 
     @Autowired private lateinit var applicationService: ApplicationService
     @Autowired private lateinit var hankeService: HankeService
-    @Autowired private lateinit var authorizer: ApplicationAuthorizer
+    @Autowired private lateinit var applicationAuthorizer: ApplicationAuthorizer
```

Comment:
Could you revert this change? This causes a lot of noise to the history and it's no longer required.

Reply:
You're right! Reverted the change and also BASE_URL (well, that could have been better to be the new HAKEMUKSET_BASE_URL but maybe later that change)